doc/changelog/09-coqide/11415-remove-ide-revert-all-buffers.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  Removed the "Revert all buffers" command from CoqIDE
```

Comment:
Again, I suggest adding a bit of info so that people better understand why this choice is made:

```suggestion
  Removed the "Revert all buffers" command from CoqIDE which had been broken for a long time
```